scripts/automation/release/version_patcher.py

```diff
@@ -0,0 +1,77 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+#pylint: disable=line-too-long
+
+import os
+import sys
+import fileinput
+
+class VersionPatcher(object):
```

Comment:
Ideally, I don't want to go back and modify all our setup.py files for this change. Changing it affects nightly builds also as that does similar version modifications as here.

I have read the link. This looks like method 4 here https://packaging.python.org/single_source_version/.
The version patcher would still be required as it would need to modify the `VERSION`/`build.info` file still right? Instead, here I modify `setup.py`.